### PR TITLE
Remove unsupported PHP and SF bundle versions

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 ### Removed
 ### Security
+- Updated dependencies (rooka/client, symfony/framework-bundle)
```
Comment (dbu) [inline changelog.md]: s/rooka/rokka ?

File: composer.json [hunk 2]
```diff
@@ -30,10 +30,10 @@
...
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "rokka/client": "^1.6.0",
-        "rokka/client-cli": "^1.6.1",
-        "symfony/framework-bundle": "^2.7 || ^3.2.2 || ^4.0"
+        "php": "^7.0",
+        "rokka/client": "^1.10",
+        "rokka/client-cli": "^1.8",
+        "symfony/framework-bundle": "^3.4 || ^4.2"
```
Comment (dbu) [inline composer.json]: i would do 3.4.26 to get rid of the security warning.

**Review by dbu — COMMENTED** [review]
apart from some nitpicks, i am happy with this.

we should look at symfony 5 support asap after this is merged. should not be overly hard i guess, we are not that deep into symfony.

File: composer.json (hunk 2 above)
Comment (thePanz) [reply composer.json]: Fixed!

File: changelog.md (hunk 1 above)
Comment (thePanz) [reply changelog.md]: Fixed!

**Comment by thePanz** [comment]
> we should look at symfony 5 support asap after this is merged. should not be overly hard i guess, we are not that deep into symfony.

Another thing we should remove is the usage of [dunglas/symfony-lock](https://packagist.org/packages/dunglas/symfony-lock) in our tests, as it has been abandoned and replaced by SF Flex

**Comment by dbu** [comment]
this is how i did that for httplugbundle: https://github.com/php-http/HttplugBundle/pull/367

**Comment by thePanz** [comment]
@dbu done in : https://github.com/rokka-io/rokka-client-bundle/pull/17

**Pull request merged**